proto/api.proto

```diff
@@ -379,39 +379,50 @@ service Api {
     };
   }
 
-  rpc GetPerpPositions(GetPerpPositionsRequest)  returns (GetPerpPositionsResponse) {
+  rpc PostCancelPerpOrder(PostCancelPerpOrderRequest) returns (PostCancelPerpOrderResponse) {
     option (google.api.http) = {
-      get: "/api/v1/trade/perp/positions"
+      post: "/api/v1/drift/perp/cancel"
```

Comment:
Yeah, most of these should be `/trade/`, not `/drift`. There is one possible exception though, which is the `PostCreateUser(PostCreateUserRequest)` and `GetUser(GetUserRequest)`, which I briefly discussed with @tinystarinagalaxy that it might be better for that to be `/drift`. You guys know better than me though.